nncf/experimental/openvino_native/quantization/quantizer_parameters.py

```diff
@@ -0,0 +1,218 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Optional, Tuple
+from dataclasses import dataclass
+import numpy as np
+
+from nncf.common.quantization.structs import QuantizationMode
+from nncf.common.quantization.structs import QuantizerConfig
+from nncf.common.tensor_statistics.statistics import MinMaxTensorStatistic
+
+
+@dataclass
+class OVQuantizerLayerParameters:
+    """
+    Class handles FakeQuantize layer attributes.
+
+    :param input_low: Tensor with minimum limit for input value.
+    :param input_high: Tensor with maximum limit for input value.
+    :param output_low: Tensor with minimum quantized value.
+    :param output_high: Tensor with maximum quantized value.
+    :param levels: Number of quantization levels.
+    """
+    input_low: np.ndarray
+    input_high: np.ndarray
+    output_low: np.ndarray
+    output_high: np.ndarray
+    levels: int
+
+
+def compute_levels(quantizer_config: QuantizerConfig, is_weights: bool) -> int:
```

Comment:
All of these transformation are applicable for each backend and should be part of common post-training algorithm. As we discussed we need to introduce common representation of FQ parameters. I propose to use NNCF representation as internal representation for FQ parameters in the common algorithms and convert it to the backend specific representation. For example:
- asymmetric quantization: `input_low`, `input_range` 
- symmetric quantization: `scale` (`input_low == 0`, `scale = input_range` )

I propose to handle it in the another PR. @l-bat, could you file a ticket on this?

Reply:
ticket: 100805